data/json/items/armor/gloves.json

```
@@ -780,10 +780,10 @@
     "color": "white",
     "covers": [ "hands" ],
     "coverage": 50,
-    "encumbrance": 5,
+    "encumbrance": 4,
     "warmth": 20,
     "material_thickness": 1,
-    "flags": [ "OVERSIZE", "ALLOWS_NATURAL_ATTACKS" ]
+    "flags": [ "SKINTIGHT", "OVERSIZE", "COMPACT", "ALLOWS_NATURAL_ATTACKS" ]
```

Worth noting that leather and fur hand wraps are still normal layer, so don't change the layer for these if you aren't changing those too.
```suggestion
    "flags": [ "OVERSIZE", "COMPACT", "ALLOWS_NATURAL_ATTACKS" ]
```